### PR TITLE
Survey Windows users in CodeLens Run 

### PR DESCRIPTION
**Pull request by mxsdev**
Solves #448

### CODE REVIEW

File: src/extension/survey.ts [hunk 1]
```diff
@@ -148,3 +148,57 @@ export class WinDefaultShell implements Disposable {
...
+    await this.#done()
+  }
+
+  async #undo() {
+    await this.context.globalState.update(SurveyWinCodeLensRun.#id, false)
+  }
+
+  async #done() {
+    await this.context.globalState.update(SurveyWinCodeLensRun.#id, true)
+  }
```
Comment (christian-bromann) [inline src/extension/survey.ts:201]: These don't seem to be used at all 🤔

File: src/extension/survey.ts [hunk 2]
```diff
@@ -148,3 +148,57 @@ export class WinDefaultShell implements Disposable {
...
+      case 'OK': {
+        TelemetryReporter.sendTelemetryEvent('survey.WinCodeLensRun', { })
+      } break
+
+      case 'Dismiss': {
+        return
+      }
+    }
+
+    await this.#done()
```
Comment (sourishkrout) [inline src/extension/survey.ts:192]: one of them is used here

**Review by sourishkrout — APPROVED** [review]
👍 LGTM. I suppose checking for `wsl` is moot since it will show up under `linux` not `win32`.

File: src/extension/survey.ts (hunk 1 above)
Comment (mxsdev) [reply src/extension/survey.ts:201]: Oh yeah, `#undo()` is not needed anymore I'll eliminate it

**Pull request merged**